src/search3_httpd_handlers.erl

```diff
@@ -0,0 +1,16 @@
+-module(search3_httpd_handlers).
+
+-export([
+    url_handler/1,
+    db_handler/1,
+    design_handler/1
+]).
+
+url_handler(_) -> no_match.
+
+design_handler(<<"_search">>) -> fun search3_httpd:handle_search_req/3;
```

Comment:
noting we'll need _search_info, _search_cleanup and  _search_analyze at some point.